front_end/panels/react_devtools/ReactDevToolsView.ts

```diff
@@ -94,38 +91,79 @@ export class ReactDevToolsViewImpl extends UI.View.SimpleView {
       send: (event, payload): void => this.sendMessage(event, payload),
     };
 
-    // To use the custom Wall we've created, we need to also create our own "Bridge" and "Store" objects.
-    this.bridge = ReactDevTools.createBridge(this.wall);
-    this.store = ReactDevTools.createStore(this.bridge);
-
     // Notify backend if Chrome DevTools was closed, marking frontend as disconnected
-    window.addEventListener('beforeunload', () => this.bridge.shutdown());
+    window.addEventListener('beforeunload', () => {
+      console.debug('[ReactDevToolsView] beforeunload');
+      this.bridge?.shutdown()
+    });
 
     SDK.TargetManager.TargetManager.instance().addModelListener(
       ReactDevToolsModel,
-      ReactDevToolsModelEvents.MessageReceived,
-      this.onMessage,
+      ReactDevToolsModelEvents.Initialized,
+      this.onInitialized,
       this,
     );
     SDK.TargetManager.TargetManager.instance().addModelListener(
       ReactDevToolsModel,
-      ReactDevToolsModelEvents.Initialized,
-      this.initialize,
+      ReactDevToolsModelEvents.Destroyed,
+      this.onDestroyed,
       this,
     );
     SDK.TargetManager.TargetManager.instance().addModelListener(
-      SDK.RuntimeModel.RuntimeModel,
-      SDK.RuntimeModel.Events.ExecutionContextDestroyed,
-      this.onExecutionContextDestroyed,
+      ReactDevToolsModel,
+      ReactDevToolsModelEvents.MessageReceived,
+      this.onMessage,
       this,
     );
+
+    this.renderLoader();
+  }
+
+  private onInitialized({data: executionContext}: ReactDevToolsInitializedEvent): void {
+    console.debug('[ReactDevToolsView] onInitialized');
+
+    this.clearLoader();
+
+    this.executionContext = executionContext;
```

Comment:
You are not using it anywhere yet, can you remove it?

Reply:
I use it in my next commit:
https://github.com/facebookexperimental/rn-chrome-devtools-frontend/commit/8262fb9756c5c289c0a349f077c0d230956c915e

Also passing it, helps to follow the code and know what context are we looking at at the moment.

It's not ideal to "just have it there not being used" but it's a useful thing to have even if not used anyway.

Reply:
> but it's a useful thing to have even if not used anyway.

How? I won't push for removing it from this PR, but that's just a deadcode, considering you are not going to merge next PR right away.

Reply:
Useful to see the relationships between the panel and context more clearly and to have the context available to do stuff if needed.